src/components/playground/runtime.js

```diff
@@ -8,12 +8,15 @@ export class Runtime {
     this.version = version;
     this.core = null;
     this.node = null;
+    this.el = null;
   }
 
   async init() {
     const pkg = await window.awaitImport(`https://cdn.skypack.dev/@elemaudio/web-renderer@^${this.version}`);
+    const coreLib = await window.awaitImport(`https://cdn.skypack.dev/@elemaudio/core@^${this.version}`); 
     const WebRenderer = pkg.default;
 
+    this.el = coreLib.el;
```

Comment:
Added `coreLib` import for version matching, and `this.el` to the runtime constructor. Great suggestion 👌